src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+#H2 Database Configuration
+spring.datasource.url=jdbc:h2:mem:surveydb
+spring.h2.console.enabled=true
+#Allow H2 Console access on Heroku
```

Comment:
This seems to be apply to all instance, whether local or Heroku by the look of the name and corresponding documentation? if that's the case I think "on Heroku" should be changed

Reply:
Since the database is in memory having the same name should not be an issue. Giving no name causes H2 to generate a random name like "jdbc:h2:mem:3a46ad2f-4dc0-43e4-8950-c5805dce89e4" for the database. 

Once we move to a persistent database we can  externalize the configuration like [this](https://devcenter.heroku.com/articles/connecting-to-relational-databases-on-heroku-with-java#using-the-jdbc_database_url-in-a-spring-boot-app).

Let me know if you have any more questions.

Reply:
Oh I was only referring to the comment on line 4, as line 5 does not look Heroku specific.

Reply:
Oh ok I can explain that better. spring.h2.console.settings.web-allow-others=true is only applicable for remote instances. So the setting does not affect local testing.